src/uu/du/src/du.rs

```diff
@@ -608,7 +572,7 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         }
     };
 
-    let block_size = read_block_size(matches.value_of(options::BLOCK_SIZE));
+    let block_size = u64::try_from(read_block_size(matches.value_of(options::BLOCK_SIZE))).unwrap();
```

Comment:
There are a lot of `usize -> u64` conversions. What was your reason for using `usize` and not `u64`?